Root.js

```diff
@@ -1,24 +1,25 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, {useCallback, useEffect, useState} from 'react';
 
-import Storybook from "./storybook";
-import App from "./App";
+// import Storybook from './storybook';
+import App from './App';
 
 const Root = () => {
   const [storybookActive, setStorybookActive] = useState(false);
   const toggleStorybook = useCallback(
     () => setStorybookActive(active => !active),
-    []
+    [],
   );
 
   useEffect(() => {
     if (__DEV__) {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const DevMenu = require("react-native-dev-menu");
-      DevMenu.addItem("Toggle Storybook", toggleStorybook);
+      const DevMenu = require('react-native-dev-menu');
+      DevMenu.addItem('Toggle Storybook', toggleStorybook);
     }
   }, [toggleStorybook]);
 
-  return storybookActive ? <Storybook /> : <App />;
+  // return storybookActive ? <Storybook /> : <App />;
+  return storybookActive ? <App /> : <App />;
```

Comment:
This does not do anything

Reply:
I guess just leave it as it is because even this commit isn't meant to be in this branch but we needed to fix something to do with asynchronous issue.

Reply:
You could comment on the top commit if necessary because there where the changes really are.